proposals/lip-0060.md

```diff
@@ -116,7 +116,7 @@ The block header is processed as follows:
   * Check that the block header follows the block header schema.
   * The value `b.header.version` can be any `uint32` integer.
```

Comment:
`b.header.version` is always some `uint32` integer.

Reply:
I think this is just a way of saying there are no additional validations.